irispy/spectrogram.py

```diff
@@ -72,3 +74,80 @@ def __init__(self, seq):
     def __getitem__(self, item):
         # slice time here
         return cu.get_cube_from_sequence(self.seq, item)
+
+class IRISSpectrogramSequence(SpectrogramSequence):
+    """A SpectrogramSequence for IRIS data including additional functionalities."""
+    def to_counts(self):
+        """Converts data and uncertainty to photon count units."""
+        for i, cube in enumerate(self.data):
+            if "FUV" in cube.meta["detector type"]:
+                DN_unit = iris_tools.DN_UNIT["FUV"]
+            elif cube.meta["detector type"] == "NUV":
```

Comment:
these two conditionals are inconsitently written

Reply:
Yes, that's intentional.  The FUV types are given as FUV1 and FUV2 but both correspond to the same DN unit.  However, there is only one NUV type so I check it more rigorously.